feed-integration-webapp/src/main/java/org/bahmni/module/feedintegration/crater/CraterLogin.java

```diff
@@ -17,19 +19,30 @@
 
 @Component
 @Scope("singleton")
+@PropertySource("/crater.properties")
 public class CraterLogin {
 
     private String token;
 
+    @Value("${crater.url}")
+    String url;
+    @Value("${crater.device_name}")
+    String device_name;
+
+    @Value("${crater.username}")
+    String username;
+
+    @Value("${crater.password}")
+    String password;
+
     public CraterLogin() {
         try {
-            CraterProperties properties = CraterProperties.getInstance();
             CloseableHttpClient httpclient = HttpClients.createDefault();
```

Comment:
not closing httpclient connection. Why not use "try clause with resources" for any closeable interface?